src/gui/sdlmain.cpp

```diff
@@ -1387,7 +1387,15 @@ static bool LoadGLShaders(const char *src, GLuint *vertex, GLuint *fragment) {
 	return "";
 }
 
-// "flexible" shaders properly handle window-resizing and NPOT textures
+[[maybe_unused]] static bool is_sharp_shader()
+{
+	const std::array<std::string, 3> sharp_shader_names{{
+	        "sharp",
+	        "default",
+	}};
```

Comment:
The `std::string` prevents making this compile-time, but switching to:

``` c++
	constexpr std::array<std::string_view, 2> sharp_shader_names{{
	        "sharp",
	        "default",
	}};
```

(If you're at it, can you also upgrade the `flexible_shader_names` array below?)

Reply:
Done.

Reply:
(Oh, try `string_view` instead of `string` for th arrays' value types)